### PR TITLE
add docs for kill auto analyze

### PR DESCRIPTION
**Pull request by xuyifangreeneyes**
### What is changed, added or deleted? (Required)

Add description on how to kill auto analyze

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/31724

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by easonn7** [comment]
LGTM

**Comment by xuyifangreeneyes** [comment]
LGTM

File: statistics.md [hunk 1]
```diff
@@ -326,7 +326,33 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
+
+    ```sql
+    KILL TIDB ${id};
+    ```
+
+    `${id}` 为上一步中查询得到的后台 `ANALYZE` 任务的 `ID`。
+
+    > **注意：**
+    >
+    > 只有当使用客户端连接到执行后台 `ANALYZE` 任务的 TiDB 实例时，执行 `KILL` 语句才能杀死后台的 `ANALYZE` 任务。如果使用客户端连接到其他 TiDB 实例或者客户端和 TiDB 中间有代理，`KILL` 语句不能杀死后台的 `ANALYZE` 任务。更多信息，请参考 [`KILL [TIDB]`](/sql-statements/sql-statement-kill.md)。
```
Comment (shichun-0415) [inline statistics.md:355]:
```suggestion
    > 只有当使用客户端连接到执行后台 `ANALYZE` 任务的 TiDB 实例时，执行 `KILL` 语句才能终止后台的 `ANALYZE` 任务。如果使用客户端连接到其他 TiDB 实例，或者客户端和 TiDB 中间有代理，`KILL` 语句不能终止后台的 `ANALYZE` 任务。更多信息，请参考 [`KILL [TIDB]`](/sql-statements/sql-statement-kill.md)。
```

File: statistics.md [hunk 2]
```diff
@@ -326,7 +326,33 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
+
+从 TiDB v6.0 起，TiDB 支持通过 `KILL` 语句终止正在后台运行的 `ANALYZE` 任务。如果发现正在后台运行的 `ANALYZE` 任务消耗大量资源影响业务，你可以通过以下步骤终止该 `ANALYZE` 任务：
+
+1. 执行以下 SQL 语句获得正在执行后台 `ANALYZE` 任务的 TiDB 实例地址和任务 `ID`：
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SELECT ci.instance as instance, cp.id as id FROM information_schema.cluster_info ci, information_schema.cluster_processlist cp WHERE ci.status_address = cp.instance and ci.type = 'tidb' and cp.info like 'analyze table %' and cp.user = '' and cp.host = '';
+    ```
```
Comment (chrysan) [inline statistics.md:339]: 为什么不是 show processlist? 是因为权限问题吗？

Comment (xuyifangreeneyes) [reply statistics.md:339]: show processlist 只能显示 instance-level 的 session。显示 cluster-level 的 session 需要查 information_schema.cluster_processlist，但这个表的 instance 字段显示的是 status address （一般端口 10080 的那个），用户用 mysql 登陆 tidb 的地址是端口 4000 的那个，所以跟 information_schema.cluster_info 做了一个 join。另外 cp.user = '' 和 cp.host = '' 是为了区分手动 analyze 和 auto analyze。

Comment (qiancai) [reply statistics.md:339]: @xuyifangreeneyes PTAL

**Review by shichun-0415 — APPROVED** [review]

**Review by chrysan — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**